examples/graphbolt/rgcn/download.py

```diff
@@ -0,0 +1,686 @@
+                "num_classes": 2983,
+                "validation_set": [
+                    {
+                        "data": [
+                            {
+                                "in_memory": in_memory,
+                                "path": "set/validation_indices_2983.npy",
+                                "name": "seeds",
+                                "format": "numpy",
+                            },
+                            {
+                                "in_memory": in_memory,
+                                "path": "set/validation_labels_2983.npy",
+                                "name": "labels",
+                                "format": "numpy",
+                            },
+                        ],
+                        "type": "paper",
+                    }
+                ],
+                "name": "node_classification_2K",
+                "train_set": [
+                    {
+                        "data": [
+                            {
+                                "in_memory": in_memory,
+                                "path": "set/train_indices_2983.npy",
+                                "name": "seeds",
+                                "format": "numpy",
+                            },
+                            {
+                                "in_memory": in_memory,
+                                "path": "set/train_labels_2983.npy",
+                                "name": "labels",
+                                "format": "numpy",
+                            },
+                        ],
+                        "type": "paper",
+                    }
+                ],
+                "test_set": [
+                    {
+                        "data": [
+                            {
+                                "in_memory": in_memory,
+                                "path": "set/test_indices_2983.npy",
+                                "name": "seeds",
+                                "format": "numpy",
+                            },
+                            {
+                                "in_memory": in_memory,
+                                "path": "set/test_labels_2983.npy",
+                                "name": "labels",
+                                "format": "numpy",
+                            },
+                        ],
+                        "type": "paper",
+                    }
+                ],
+            },
+        ],
+        "feature_data": [
+            {
+                "domain": "node",
+                "name": "feat",
+                "format": "numpy",
+                "in_memory": in_memory,
+                "path": "data/paper_feat.npy",
+                "type": "paper",
+            },
+            {
+                "domain": "node",
+                "name": "label",
+                "format": "numpy",
+                "in_memory": in_memory,
+                "path": "data/paper_label_19.npy",
```

Comment:
Why is there no 2K labels here? Also, are labels also added as features to the dataset?

Reply:
I will add the 2983 now, and yes, I added it as feature data. I am not sure where to add the label

Reply:
The labels are part of the tasks?

Reply:
Yes. But the label file is already processed into the three sets, so the original whole label file is not needed

Reply:
@frozenbugs Do we need to add labels as features?